__tests__/app.test.js

```diff
@@ -73,44 +75,85 @@ describe('GET', () => {
             author: "icellusedkars",
             title: "Eight pug gifs that remind me of mitch",
             article_id: 3,
-            topic: 'mitch',
-            created_at: '2020-11-03T09:12:00.000Z',
+            topic: "mitch",
+            created_at: "2020-11-03T09:12:00.000Z",
             votes: 0,
           });
-      })
+        });
+    });
+  });
+
+  describe("/api/articles/:article_id/comments", () => {
+    it("returns the requested article selected by id", () => {
+      const ARTICLE_ID = 5;
+      return request(app)
+        .get(`/api/articles/${ARTICLE_ID}/comments`)
+        .expect(200)
+        .then(({ body: { comments } }) => {
+          expect(comments).toBeInstanceOf(Array);
+          comments.forEach((comment) => {
+            expect.objectContaining({
+              comment_id: expect.any(Number),
+              votes: expect.any(Number),
+              created_at: expect.any(String),
+              author: expect.any(String),
+              body: expect.any(String),
+            });
```

Comment:
I would probably want to also assert the review_id here - otherwise, can we be sure the right comments have been sent?